int8                       type 
int64                      id
geometry_msgs/PoseStamped  pose
